app/use_cases/post_enrolment_authorisation.py

```diff
@@ -0,0 +1,37 @@
+from pydantic import BaseModel
+
+from app.repositories.enrolment_repo import EnrolmentRepo
+from app.requests.enrolment_authorisation_request import EnrolmentAuthorisationRequest
+from app.responses import ResponseFailure, ResponseSuccess, SuccessType
+
+
+class PostEnrolmentAuthorisationRequest(BaseModel):
```

Comment:
It might be confusing to have "request" in a usecase name. Please change it. I think `EnrolmentAuthorisation` would be enough.